kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -258,34 +306,8 @@ public void run() {
     }
   }
 
-  private boolean launchEngineOpCompleted() {
-    TGetOperationStatusReq opStatusReq = new TGetOperationStatusReq(launchEngineOpHandle);
-    try {
-      return client.GetOperationStatus(opStatusReq).getOperationCompleted() != 0;
-    } catch (TException e) {
-      return true;
-    }
-  }
-
-  private List<String> fetchEngineLogs() throws SQLException {
-    TFetchResultsReq fetchResultsReq = new TFetchResultsReq(launchEngineOpHandle,
-      TFetchOrientation.FETCH_NEXT, fetchSize);
-    fetchResultsReq.setFetchType((short) 1);
-
-    List<String> logs = new ArrayList<>();
-    try {
-      TFetchResultsResp tFetchResultsResp = client.FetchResults(fetchResultsReq);
-      RowSet rowSet = RowSetFactory.create(tFetchResultsResp.getResults(), this.getProtocol());
-      for (Object[] row: rowSet) {
-        logs.add(String.valueOf(row[0]));
-      }
-    } catch (TException e) {
-      throw new SQLException("Error building result set for query log", e);
-    }
-    return Collections.unmodifiableList(logs);
-  }
-
-  private void executeInitSql() throws SQLException {
+  public void executeInitSql() throws SQLException {
```

Comment:
Why public this method?

Reply:
invoke it in beeline after launching get engine log thread